lib/eslint/flat-eslint.js

```diff
@@ -281,7 +282,42 @@ async function loadFlatConfigFile(filePath) {
 
     debug(`Config file URL is ${fileURL}`);
 
-    return (await import(fileURL)).default;
+    const mtime = (await fs.stat(filePath)).mtime.getTime();
```

Comment:
I wonder if all of this is strictly necessary. Can't we just append `?time=${new Date().getTime()}` to the URL and forgo needing to check the actual modification time?

Reply:
That would work, but every time we call `loadFlatConfigFile` a new instance would be created and cached forever. I left a note on why this solution might be problematic here:

https://github.com/eslint/eslint/blob/b82bd5618f396b02995e0d912c36dab54cdf048a/lib/eslint/flat-eslint.js#L297-L299

Reply:
Ah, totally missed that. Makes sense. :+1: